sklearn/inspection/_plot/tests/test_boundary_decision_display.py

```diff
@@ -232,7 +314,7 @@ def fit(self, X, y):
 
     clf = MyClassifier().fit(X, y)
 
-    with pytest.raises(ValueError, match=msg):
+    with pytest.raises(AttributeError, match=msg):
```

Comment:
This behavior change could be considered a breaking change... or a fix. I am not sure. If the latter, it should at least be documented in the changelog: please add a second entry with the `|Fix|` label.

Or alternatively we could make it such that we still raise `ValueError` instead of `AttributeError` to avoid any necessary disruption but I cannot find in the new code where this happens.

I think I am in favor of keeping the `ValueError` if it's not making the code too convoluted.

Reply:
I check and we already have a `try ... except` around `_get_response_values` and thus I think that we can re-raise the error without too convoluted code. If the `try ... except` was not already there, I would have been in favor of letting the code as-is.